src/user-guide/installing-workflows.rst

```diff
@@ -0,0 +1,370 @@
+.. _Installing-workflows:
+
+Installing Workflows
+====================
+
+Cylc commands identify workflows via their names, which are relative path names
+under the :term:`cylc-run directory`, ``~/cylc-run/`` by default.
+
+Workflows can be grouped together under sub-directories. E.g.:
+
+.. code-block:: console
+
+   $ cylc scan --state=all --name nwp --format=tree
+   nwp
+   ├─oper
+   │ ├─ region1
+   │ └─ region2
+   └─test
+      └─ region1
+
+This chapter will demonstrate how to install a workflow from an arbitrary
+location, called a :term:`source directory`.
+``cylc install`` will create a new directory in the :term:`cylc-run directory`
+for each installation of a workflow.
+
+.. _Install-Workflow:
+
+The Cylc Install Command
+------------------------
+
+Workflow names can be installed with the ``cylc install`` command,
+which creates the :term:`run directory` structure and some service
+files underneath it. Otherwise, ``cylc play`` will do this at workflow start
+up.
+
+Once you have written your workflow, you can have Cylc install the workflow for
+you, using the ``cylc install`` command.
+
+.. _Command Line Options:
+
+Command Line Options
+--------------------
+
+The following commands, executed from :term:`source directory` e.g.
+``~/cylc-sources/my-flow`` result in the following installations.
+
+
++--------------------------------------+--------------------------------------+
+| Command                              | Results in Installation in Directory |
++======================================+======================================+
+| cylc install                         |    ~/cylc-run/my-flow/run1           |
++--------------------------------------+--------------------------------------+
+| cylc install --no-run-name           |    ~/cylc-run/my-flow                |
++--------------------------------------+--------------------------------------+
+| cylc install --run-name=new-run      |    ~/cylc-run/my-flow/new-run        |
++--------------------------------------+--------------------------------------+
+| cylc install --flow-name=new-name    |    ~/cylc-run/new-name/run1          |
++--------------------------------------+--------------------------------------+
+
+Any of the above commands may be run from anywhere on the file system with the
+addition of the option ``--directory=PATH/TO/SOURCE/DIRECTORY``, alternatively
+use ``-C PATH/TO/SOURCE/DIRECTORY``.
+
+Numbered Runs
+^^^^^^^^^^^^^
+
+By default, cylc install will install the workflow found in the current working
+directory into ``~/cylc-run/$(basename $PWD)/runN``, where runN = run1, run2, 
+run3,...
+
+``cylc install`` will automatically increment the run number of each install,
+provided the options ``--no-run-name`` or ``--run-name`` are not used. See
+:ref:`Command Line Options` for example behaviour.
+
+For convenience, a symlink to the highest (latest) numbered run will be created
```

Comment:
```suggestion
For convenience, a symlink to the most recent (highest numbered) run will be created
```
I think that it's the most recent bit that is important to the sense of the sentence, and the "highest numbered" bit is an explanation of how Cylc Knows.